phylorthology/README.md

```diff
@@ -0,0 +1,11 @@
+# phylorthology
+
+## Information & purpose
+
+This repository holds the test dataset and the input file for the [phylorthology pipeline](https://github.com/Arcadia-Science/phylorthology).
+
+The [samplesheet CSV](samplesheet_test.csv) includes all the file paths and the metadata for the test proteomes. The [nextflow_parameters.json](nextflow_parameters.json) file specifies all the inputs to the [phylorthology pipeline](https://github.com/Arcadia-Science/phylorthology) and can be used via the `-params-file` CLI option.
+
+## Test dataset curation
+
+TODO: [Austin Patton](https://github.com/austinhpatton) will include more details around the curation of this dataset.
```

Comment:
I'm going to leave this as a TODO for now - we can figure this out in more detail once we work out what size of a dataset we should include as a test dataset (and how rigorous the results of that test should be).